src/rules/first-name-rules.ts

```diff
@@ -1,6 +1,5 @@
 export const FirstNameRules: Array<Function> = [
-  v => !!v || 'A first name is required',
-  v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
-  v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+  v => !!v || 'First name is required',
+  v => /^(?=.*\d)|(?=.*[a-z])|(?=.*[A-Z])/.test(v) || 'First name is required', // Contains at least a single character
```

Comment:
Isn't this obsolete, since the value is trimmed so it must be something?
Or is this to limit the valid character set?

Reply:
Trim will remove the space before/after the strings input but if a user copy pasted a big block of white space in, it wouldn't be caught by our initial rule. So yea, this is to confirm there is an actual character in there. 
Seems like white space is considered a form of character.

Reply:
Can you give me some test steps to reproduce this? I am doing something similar for Transactional Folio Number and trim seems to work fine 🤷‍♂️

Reply:
I found i could do it just by hitting space repeatedly.

Reply:
Same implementation?
ie `v-model.trim="someStringValue"` ?

Reply:
I think it's a Vuetify bug! I was able to paste enough spaces to exceed my 50 chars, but still the correct value (with no spaces) is mutated in the store.

Reply:
This fixes it for me:
```
export const FolioNumberRules: Array<Function> = [
  v => (!v || v.trim().length <= 50) || 'Cannot exceed 50 characters' // maximum character count
]
```

Reply:
> Same implementation? ie `v-model.trim="someStringValue"` ?

Yeah, I thought you copied that from me. Great minds think alike!

Reply:
Ahh yea, most likely! 
I wonder where/when the trim occurs.

Reply:
Based on my testing, the v-model is trimmed before being updated, but the component still processes its rules against the internal value (that is in the input field).

Reply:
I'll try updating my rules to match yours, it's cleaner than way if it works all the same!

Reply:
updated, much better, thank you!